.github/workflows/python-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Python CI
+
+on: push
+
+jobs:
+  test:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [2.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      env:
+        PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
+      run: |
+        sudo apt-get install postgresql-10-postgis-2.3 postgresql-10-postgis-scripts
+        pip install -r requirements.txt -r test_requirements.txt
+        pip install .
+    - name: Test with pytest (full)
+      if: github.ref == 'refs/heads/master'
+      run: |
+        py.test -v --tb=short tests
+    - name: Test with pytest (fast)
+      if: github.ref != 'refs/heads/master'
+      run: |
+        py.test -m "not slow" -v --tb=short tests
```

Comment:
I don't think any tests in this repository are marked "slow". Am I missing something?